### PR TITLE
Don't panic in init when bpf programs don't log

### PR DESCRIPTION
**Pull request by ajwerner**
This was broken in d6cb1a16ad0f8df483e2234fb01ab55bdbeaa8b8.

### CODE REVIEW

File: aya-log/src/lib.rs [hunk 1]
```diff
@@ -324,8 +327,13 @@ impl Log for DefaultLogger {
     }
 }
 
+const MAP_NAME: &str = "AYA_LOGS";
```
Comment (vadorovsky) [inline aya-log/src/lib.rs:330]: Please move that const to the top of the file.

**Review by vadorovsky — COMMENTED** [review]
Thanks! In general looks good, just one minor comment. @alessandrod do you agree?

File: aya-log/src/lib.rs (hunk 1 above)
Comment (ajwerner) [reply aya-log/src/lib.rs]: Done.

**Review by vadorovsky — APPROVED** [review]

File: aya-log/src/lib.rs [hunk 2]
```diff
@@ -326,6 +331,9 @@ impl Log for DefaultLogger {
 
 #[derive(Error, Debug)]
 pub enum Error {
+    #[error("log event array {} doesn't exist", MAP_NAME)]
+    MapMissingError,
```
Comment (alessandrod) [inline aya-log/src/lib.rs:335]: I think this should be `MapNotFound`.

We use the error suffix when forwarding an inner error, eg `MapError(map_error)` `IOError(io_error)` etc.

**Review by alessandrod — APPROVED** [review]
Nice catch! See minor nit

File: aya-log/src/lib.rs (hunk 2 above)
Comment (ajwerner) [reply aya-log/src/lib.rs]: Done.

**Review by alessandrod — APPROVED** [review]

**Pull request merged**